bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALPtoTTbar/ALP_ttbar_interference/ALP_ttbar_interference_run_card.dat

```diff
@@ -0,0 +1,274 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************
+# Running parameters
+#*******************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  100 = nevents ! Number of unweighted events requested
```

Comment:
change this to 3000, pilot run 100 is a bit small

Reply:
Okay, I think this here was just the default value in my madgraph card.
I'm no expert, but won't this be overwritten later anyways?
We would like target 10M events for each sample (signal, interference) and I will contact our b2g MC experts on this.

Reply:
No impact on production. Just needed for the pilot run so you get enough stats to have a meaningful test.

Reply:
Alright, thanks for the info, then I will change this value to 3000 in both cases